docs/installation/mac.md

```diff
@@ -384,6 +384,12 @@ The next exercise demonstrates how to do this.
         $ docker stop mysite
         $ docker rm mysite
 
+**Note**
+
+There is a [known
+issue](https://docs.docker.com/machine/drivers/virtualbox/#known-issues) that
+may cause files shared with your nginx container to not update correctly as you
+modify them on your host.
```

Comment:
Can you out this in our standard "note" format?

```Markdown
> **Note**: There is a [known issue](https://docs.docker.com/machine/drivers/virtualbox/#known-issues)
> that causes files shared with your nginx container to not update correctly as you
> modify them on your host.
```

Also, can you add the same note to the `windows.md` file?